cmd/argocd/commands/project.go

```diff
@@ -63,17 +92,130 @@ func NewProjectCreateCommand(clientOpts *argocdclient.ClientOptions) *cobra.Comm
 			errors.CheckError(err)
 		},
 	}
-	command.Flags().StringVar(&proj.ObjectMeta.Name, "name", "n", "Project name")
-	command.Flags().StringVarP(&proj.Spec.Description, "description", "", "desc", "Project description")
-	command.Flags().StringArrayVarP(&destinations, "dest", "d", []string{},
-		"Allowed deployment destination. Includes comma separated server url and namespace (e.g. https://192.168.99.100:8443,default")
+	command.Flags().StringVar(&name, "name", "n", "Project name")
```

Comment:
Sorry, I didn't notice this the first review. I think name should be a positional arg (like `cluster add` and `repo add`).